packages/terra-spacer/src/Spacer.scss

```diff
@@ -0,0 +1,325 @@
+:local {
+  .margin-top-none {
+    margin-top: 0;
+  }
+
```

Comment:
Since the fallback values for the scale are the same across all the styles, you could create scss variables and use those as the fallback.

```scss
// Define these at the top of this file
$small-minus-2: 0.2142857143rem;

// Then use them in each class
.margin-top-small-minus-2 {
  margin-top: var(--terra-spacer-margin-top-small-minus-2, $small-minus-2); // 3px
}
```

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/b2ec65a5ad9f9f5b81c9394939eb477f56b21774#diff-bbd694561d6e058afb7ec7501feecb1aR1